src/statmodels.jl

```diff
@@ -79,20 +108,48 @@ when applicable the intercept and the distribution's dispersion parameter.
 """
 dof(obj::StatisticalModel) = error("dof is not defined for $(typeof(obj)).")
 
+"""
+    ess(obj::StatisticalModel)
+
+Return the the explained sum of squares.
+"""
+ess(obj::StatisticalModel) = sum(abs2, fitted(obj) - meanresponse(obj))
+
+"""
+    rss(obj::StatisticalModel)
+
+Return the the residual sum of squares.
+"""
+rss(obj::StatisticalModel) = sum(abs2, (model_response(obj) - fitted(obj)))
+
 """
     stderr(obj::StatisticalModel)
 
 Return the standard errors for the coefficients of the model.
 """
 stderr(obj::StatisticalModel) = sqrt.(diag(vcov(obj)))
 
+"""
+    tss(obj::StatisticalModel)
+
+Return the total sum of squares.
+"""
+tss(obj::StatisticalModel) = error("tss is not defined for $(typeof(obj)).")
```

Comment:
Isn't this equivalent to `deviance`?

Reply:
The deviance of some models use `tss` in the calculation. For example,
```
struct OLS <: RegressionModel end
deviance(model::OLS) = tss(model) / dof_residual(model)
```

Reply:
Mmm, that's not what GLM does:
https://github.com/JuliaStats/GLM.jl/blob/287c668acd592c05ec90140dfa70a2887045b4f3/src/lm.jl#L46-L61

Reply:
I stand corrected. The `deviance` for non-linear models is still computed the same way (by the generalization). Since they are equivalent as far as I believe now, we can just keep `deviance`.

Reply:
Though if we add `mss` and `rss`, it would make sense to have  `tss` too, and let linear models implement it by calling `deviance` (or vice-versa).